### PR TITLE
Added Far Cry 2

### PR DESCRIPTION
**Pull request by ImperatorS79**
Testing because I do not own the game.

### CODE REVIEW

**Comment by plata** [comment]
As you couldn't test the game: can you provide some evidence that it should run with that configuration (e.g. AppDB)?

**Comment by ImperatorS79** [comment]
Latest test was for wine 2.1 ->[https://appdb.winehq.org/objectManager.php?sClass=version&iId=14238]( https://appdb.winehq.org/objectManager.php?sClass=version&iId=14238)

**Comment by ImperatorS79** [comment]
@plata Do you want me to add this as comments inside the script ?

**Comment by plata** [comment]
No. I just believe that we have to be really careful about the amount of more or less untested scripts. On the one hand, it's good to provide the script so people who would like to have it can test it. On the other hand, we shouldn't have too many scripts which just don't work.

File: Applications/Games/Far Cry 2/Steam/script.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "secur32"]);
+
+new SteamScript()
+    .name("Far Cry 2")
```
Comment (plata) [inline Applications/Games/Far Cry 2/Steam/script.js]: Shouldn't there be a ®?

File: Applications/Games/Far Cry 2/application.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+     "name": "Far Cry 2",
```
Comment (plata) [inline Applications/Games/Far Cry 2/application.json]: Shouldn't there be a ®?

**Comment by plata** [comment]
The other place is still missing.

**Pull request merged**